DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Administrator/AdministratorController.cs

```diff
@@ -35,6 +39,12 @@ ICommonService commonService
             int page = 1
         )
         {
+            // Query parameter should take priority over cookie value
+            if (!Request.Query.ContainsKey(nameof(filterBy)))
+            {
+                filterBy = Request.Cookies[AdminFilterCookieName];
+            }
+
             filterBy = FilteringHelper.AddNewFilterToFilterBy(filterBy, filterValue);

```

Comment:
I haven't quite got my head around the roles of filterValue and filterBy - would you enlighten me?

Reply:
filterBy is the full list of ╡ separated filters to be applied.
filterValue is a new filter to be appended to the filterBy string (and then applied)

There is a more in depth explanation here:
https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/19584057357/Implementing+no+JavaScript+search+sort+filter+and+paginate
https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/19577569424/Javascript+search+sort+paginate+functionality+with+js-search

Reply:
Yeah, sounds reasonable. Cheers!